packages/react/src/FilteredActionList/FilteredActionListWithModernActionList.tsx

```diff
@@ -38,6 +39,7 @@ export interface FilteredActionListProps
   onInputRefChanged?: (ref: React.RefObject<HTMLInputElement>) => void
   textInputProps?: Partial<Omit<TextInputProps, 'onChange'>>
   inputRef?: React.RefObject<HTMLInputElement>
+  message?: React.ReactNode[]
```

Comment:
```suggestion
  message?: React.ReactNode
```

The docs.json has this as `React.ReactNode`, just wanted to point this out in case one of them need to change 👀

Reply:
ye[, should be `React.ReactNode`, Thanks!